README.md

```diff
@@ -75,11 +75,12 @@ The options object can contain properties from [react router's create configurat
 
 Additionally, it can contain the following **optional** properties,
 
-- `routesFilePath`: <String> - path for the file that contains the react router routes.
+- `routesFilePath`: \<String> - path for the file that contains the react router routes.
                    react-engine uses this behind the scenes to reload the routes file in
                    cases where [express's app property](http://expressjs.com/api.html#app.set) `view cache` is false, this way you don't need to restart the server every time a change is made in the view files or routes file.
-- `renderOptionsKeysToFilter`: <Array> - an array of keys that need to be filtered out from the data object that gets fed into the react component for rendering. [more info](#data-for-component-rendering)
-- `performanceCollector`: <Function> - to collects [perf stats](#performance-profiling)
+- `renderOptionsKeysToFilter`: \<Array> - an array of keys that need to be filtered out from the data object that gets fed into the react component for rendering. [more info](#data-for-component-rendering)
+- `performanceCollector`: \<Function> - to collects [perf stats](#performance-profiling)
+- `page404`: \<Function> - This option allows you to define a custom 404 page when a URL is not matched in your routes. If left undeclared, a default 404 page, interally provided by this library, is rendered.
```

Comment:
Better to call `page404` as a `ReactComponent` than `Function` :) 
typo - `interally` -> `internally`